operator/internal/status/lokistack.go

```diff
@@ -17,10 +17,11 @@ import (
 )
 
 const (
-	messageReady              = "All components ready"
-	messageFailed             = "Some LokiStack components failed"
-	messagePending            = "Some LokiStack components pending on dependencies"
-	messageDegradedNodeLabels = "Cluster contains no nodes matching the labels used for zone-awareness"
+	messageReady                      = "All components ready"
+	messageFailed                     = "Some LokiStack components failed"
+	messagePending                    = "Some LokiStack components pending on dependencies"
+	messageDegradedNodeLabels         = "Cluster contains no nodes matching the labels used for zone-awareness"
+	messageTopologyKeyNodeLabelsEmpty = "No value for the labels used for zone-awareness"
```

Comment:
For better readability I would use the `messageDegraded` prefix for the second condition as well:

```suggestion
	messageDegradedMissingNodes   = "Cluster contains no nodes matching the labels used for zone-awareness"
	messageDegradedEmptyNodeLabel = "Node-label used for zone-awareness has an empty value"
```